planetcantile/data/generate.py

```diff
@@ -48,30 +48,46 @@ def __post_init__(self):
 with urllib.request.urlopen('https://raw.githubusercontent.com/pdssp/planet_crs_registry/main/data/result.wkts') as response:
     resp = response.read().decode(response.headers.get_content_charset())
 
-# Parse all the GeoCRSs from the list
-geocrss = []
+# Parse all the CRSs from the list
+allcrss = []
+acceptable_projections = ('Equirectangular', 'North Polar', 'South Polar')
 for wkt_str in resp.split(os.linesep + os.linesep):
-    if 'GEOGCRS' in wkt_str[:7] and 'TRIAXIAL' not in wkt_str:  # insanely hacky
-        geocrss.append(wkt_str)
+    if 'TRIAXIAL' not in wkt_str:  # insanely hacky
+        if wkt_str.startswith('GEOGCRS') or any(_ in wkt_str for _ in acceptable_projections):
+            allcrss.append(wkt_str)
 
 # Build a dynamic crss list
 crss = []
-for crs in geocrss:
+for crs in allcrss:
     crs_obj = CRS(crs)
     title = crs_obj.name
     auth = crs_obj.to_authority(min_confidence=25)
     if auth is not None:
         authority_version, code = auth
         authority, version = authority_version.split('_')
         identifier = f'{authority}_{code}_{version}'
+        geographic_crs = crs_obj.geodetic_crs
+        # get the extent, if clon == 180 we have a 0-360 longitude crs
+        extent = (0.0, -90.0, 360.0, 90.0) if 'clon = 180' in crs else (-180.0, -90.0, 180.0, 90.0)
+        # do some work to get extent from projected CRSs
+        if crs_obj.is_projected:
+            # todo: this doesn't work well with ographic geographic crss (those with inverse flattenings), fails ecq: Invalid latitude
+            # looks to be an issue where the x and y is swapped, if the inverse_flattening is not 0.0 we would have to swap
```

Comment:
Does auth or geographic_crs have a way to get the axis mapping? For example, if you run a gdalinfo you will see: `Data axis to CRS axis mapping: 1,2`. If we can get the axis mapping it is parsable to help with this (I think?)

Reply:
I am not sure if the axis order is actually swapped or not, I don't see a way of grabbing the mapping from the coordinate_system object in pyproj. There is an axis_list attribute that returns the two axes in the order they are specified but I didn't see a swap in the axis order in the CRSs themselves, just instead of easting you see a westing

Reply:
Ahh! the axis ordering of the geodetic_crs is swapped 

```
In [42]: b.coordinate_system
Out[42]:
CS[Cartesian,2],
    AXIS["westing (W)",west,
        ORDER[1],
        LENGTHUNIT["metre",1,
            ID["EPSG",9001]]],
    AXIS["(N)",north,
        ORDER[2],
        LENGTHUNIT["metre",1,
            ID["EPSG",9001]]]

In [43]: b.geodetic_crs.coordinate_system
Out[43]:
CS[ellipsoidal,2],
    AXIS["geodetic latitude (Lat)",north,
        ORDER[1],
        ANGLEUNIT["degree",0.0174532925199433,
            ID["EPSG",9122]]],
    AXIS["geodetic longitude (Lon)",west,
        ORDER[2],
        ANGLEUNIT["degree",0.0174532925199433,
            ID["EPSG",9122]]]
```

where b is IAU:69911

Reply:
woohoo! Great detective work!